Jenkinsfile

```
@@ -28,10 +28,10 @@ pipeline {
     }
 
     parameters {
-        booleanParam(name: 'DEPLOY_TO_DEV', defaultValue: false, description: 'Should this version be deployed to dev stage (by default master will deploy to int stage; other branches do not deploy at all)');
+        booleanParam(name: 'DEPLOY_TO_DEV', defaultValue: false, description: 'Should this version be deployed to dev stage (by default the develop branch will deploy to dev stage; stable branches are deployed to prod and other branches do not deploy at all)');
```

This is not entirely correct. Develop is by default deployed to production stage.

I have two suggestions:
a) "this is what you need to know"-style:
```suggestion
        booleanParam(name: 'DEPLOY_TO_DEV', defaultValue: false, description: 'Click here if you want to test a feature in the development environment prior to merge');
```
b) "explaining what this means"-style:
```suggestion
        booleanParam(name: 'DEPLOY_TO_DEV', defaultValue: false, description: 'Should this version be deployed to dev stage? (by default changes to develop and stable branches are deployed to production; feature branches are not deployed at all. However, if this is checked the build will be deployed to a dev environment for testing. This works for all branches)');
```

---

Ah! You're absolutely right. These stages mess with my brain sometimes 😄 Out of the 2 given options, my preference goes out to the `what-you-need-to-know`-style, but I do want to document the other part somewhere. I'll have a look if I can get this into the README.

---

https://github.com/zeebe-io/zeebe-cluster-testbench/blob/develop/docs/developer-handbook.md is probably a better place to document this